go/vt/callinfo/plugin_grpc.go

```diff
@@ -23,27 +23,12 @@ import (
 	"html/template"
 
 	"golang.org/x/net/context"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/peer"
 )
 
 // GRPCCallInfo returns an augmented context with a CallInfo structure,
 // only for gRPC contexts.
 func GRPCCallInfo(ctx context.Context) context.Context {
-	method, ok := grpc.Method(ctx)
-	if !ok {
-		return ctx
-	}
-
-	callinfo := &gRPCCallInfoImpl{
-		method: method,
-	}
-	peer, ok := peer.FromContext(ctx)
-	if ok {
-		callinfo.remoteAddr = peer.Addr.String()
-	}
-
-	return NewContext(ctx, callinfo)
+	return NewContext(ctx, nil)
```

Comment:
I presume this change was unintentional?

Reply:
@mcuadros - ^^^